src/background.js

```diff
@@ -137,3 +140,30 @@ const refreshOAuthToken = async ({ refreshToken }) => {
     return { success: false, response: e.message };
   }
 };
+
+const loadMemberData = async ({ username, platform, ORBIT_CREDENTIALS }) => {
```

Comment:
Would it be possible to load the member credentials inside `background.js`, and not have them be passed down by the widget?

Reply:
So, originally I tried to do this (and it would have made things a lot easier, especially since we'll need them again to add members to orbit). But in `getOrbitCredentials` we send another message to refresh the credentials, if needed, and this was routinely failing. Something to do with not being able to access the chrome runtime environment, it was a few days ago. Spent about 1/2 day trying to resolve it before just moving the credentials out of here.

Happy to go back & try resolve again, but figured this was a pragmatic solution